pages/pricing.js

```diff
@@ -412,12 +412,11 @@ export default function Premium({ user }) {
                     className="flex w-1/3 px-4"
                     aria-hidden="true"
                     style={{
-                      marginLeft: `${
-                        (tiers.findIndex((tier) => tier.mostPopular) + 1) * 33
-                      }%`,
+                      marginLeft: `${(tiers.findIndex((tier) => tier.mostPopular) + 1) * 33
```

Comment:
I don't see any changes in these line, however the indentation looks disturbed compared to original state. Please format your document(Ctrl + Shift + I in VS Code) before pushing changes.